locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -2225,15 +2247,15 @@ msgstr "Email de l’administrateur :"
 msgid "Additional info:"
 msgstr "Infos supplémentaires :"
 
-#: bookwyrm/templates/settings/site.html:83
-msgid "Allow registration:"
-msgstr "Autoriser l’enregistrement :"
-
-#: bookwyrm/templates/settings/site.html:87
-msgid "Allow invite requests:"
-msgstr "Autoriser les demandes d’invitation :"
+#: bookwyrm/templates/settings/site.html:85
+msgid "Allow registration"
+msgstr "Autoriser les inscriptions"
```

Comment:
There are other occurrences of “enregistrement” that could use some love if you really want to switch to “inscription” instead.